toolkits/springernature/packages/springernature-header/scss/10-settings/layout.scss

```diff
@@ -0,0 +1,2 @@
+$global-header-height: 40px;
+$max-content-width: 1184px;
```

Comment:
naming conventions for component variables `$component-name--variable-name`, so here would be
`$header--header-height`
`$header--content-width`